R/verify_ts.R

```diff
@@ -34,7 +34,7 @@ verify_ts <- function(data, var, checks = c('ncol', 'unitted', 'tz', 'units', 'n
       is.unitted(x)
     },
     'tz' = function(x,...) {
-      (get_units(x[,1]) == 'UTC') || 
+      (get_units(x[,1]) == '') && attr(x[,1],'tzone') == 'UTC' || 
```

Comment:
i'm never sure about order of operations on logicals - this is equal to 
```
((get_units(x[,1]) == '') && (attr(x[,1],'tzone') == 'UTC')) || 
  (all(is.na(x[,1])) && nrow(x) == 1) # allow for 1-row const tses
```
, right?